src/pkg/leaderElection.go

```diff
@@ -0,0 +1,69 @@
+package pkg
+
+import (
+	"context"
+	"github.com/rs/zerolog/log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"time"
+)
+
+var (
+	isLeader bool
+)
+
+func RunLeaderElection(client *clientset.Clientset, ctx context.Context, lockname, id, namespace string) {
+	lock := &resourcelock.LeaseLock{
+		LeaseMeta: metav1.ObjectMeta{
+			Name:      lockname,
+			Namespace: namespace,
```

Comment:
```suggestion
			Namespace: lockNamespace,
```

just to be very clear to the person calling `RunLeaderElection`